### PR TITLE
storage-network: add doc

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/4081
https://github.com/longhorn/longhorn/issues/2285

### CODE REVIEW

**Review by derekbit — APPROVED** [review]
LGTM

File: content/docs/1.3.0/advanced-resources/deploy/storage-network.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+You can verify by creating a simple DaemonSet and try ping between pods.
+
+### Setting Storage Network During Longhorn Installation
+Follow the [Customize default settings](../customizing-default-settings/) to set Storage Network by changing the value for the `storage-network` default setting
+
+> **Warning:** Longhorn instance-manager will not start if the Storage Network setting is invalid.
+>
+> You can check the events of the instance-manager Pod to see if it is related to an invalid NetworkAttachmentDefintion with `kubectl -n longhorn-system describe pods -l longhorn.io/component=instance-manager`.
+>
+> If this is the case, provide a valid `NetworkAttachmentDefinition` and re-run Longhorn install.
```
Comment (innobead) [inline content/docs/1.3.0/advanced-resources/deploy/storage-network.md:34]: Should we change to this?

```suggestion
> If this is the case, provide a valid `NetworkAttachmentDefinition`, then Longhorn will automatically restart all related managed pods.
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one comment.

File: content/docs/1.3.0/advanced-resources/deploy/storage-network.md (hunk 1 above)
Comment (c3y1huang) [reply content/docs/1.3.0/advanced-resources/deploy/storage-network.md:34]: This is for fresh installing Longhorn with the storage network in the default setting. I think re-running the install is more straightforward.

**Review by innobead — APPROVED** [review]

**Pull request merged**